packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -31,7 +31,7 @@
 %endif
 
 Name:           ecs-init
-Version:        1.22.0
+Version:        1.23.0
 Release:        4%{?dist}
```

Comment:
The Release tag should change to `1` with the rollover to a new Version.

Reply:
Thanks for pointing out. Changed it to 1